File: php/examples/get_filing_methods_schemas.php
<?php
require_once 'base_request.php';

// Example of GET /filing-methods/schemas

$debug = true;
$company_id   = $_ENV['COMPANY_ID'];
$method_id = $_ENV['FILING_METHOD_ID'];
$request_data = '';
$request_params = ['company_id' => $company_id, 'filing_method_id' => $method_id];

send_request('GET', '/filing-methods/schemas', $request_params, $request_data);

File: csharp/examples/GetFilingMethodsSchemas.cs
using System;

namespace Examples.examples
{
    // Example of GET /filing-methods/schemas
    
    class GetFilingMethodsSchemas : BaseRequest
    {
        private string _companyId;
        private string _methodId;

        public GetFilingMethodsSchemas()
        {
            dotenv.net.DotEnv.Load();
            _companyId = Environment.GetEnvironmentVariable("COMPANY_ID");
            _methodId = Environment.GetEnvironmentVariable("FILING_METHOD_ID");
            Console.WriteLine($"GetFilingMethodsSchemas: company_id={_companyId} filing_method_id={_methodId}");
        }

        public GetFilingMethodsSchemas(string companyId, string filingMethodId)
        {
            _companyId = companyId;
            _methodId = filingMethodId;
        }

        public override string SendRequest()
        {
            string path = "filing-methods/schemas";
            string queryParams = $"?company_id={_companyId}&filing_method_id={_methodId}";
            return GetRequest(path + queryParams);
        }
    }
}
